pkg/backend/backend.go

```diff
@@ -358,3 +362,41 @@ func (c *backendClient) GetStackResourceOutputs(
 	}
 	return pm, nil
 }
+
+// CreateStackOptions provides options for stack creation. At present,
+// options only apply to the Service.
+type CreateStackOptions interface {
+	// Teams returns the list of teams who should have access to
+	// the newly created stack. This method is only approperate
+	// for backends which support teams (i.e. the Service).
+	Teams() []string
+}
```

Comment:
I definitely disagree here. But instead of relitigating the merits, I'll make the change in a follow-on PR.

Since your objection isn't a matter of correctness and given that @justinvp has asked we get this feature in ASAP, 
I'm afraid I have to insist we leave it as-is in _this_ PR, but I'll open an issue and create a _follow-on_ PR today to change the type from an interface into a struct.

Reply:
I agree with the feedback to use a simple struct here (`*CreateStackOptions`). I'd expect it'd be a pretty quick change to make as part of this PR today. No need to do it as a follow-up.